ui/console/src/main/scripts/plugins/metrics/plugins/metrics/ts/addUrlPage.ts

```diff
@@ -194,7 +194,8 @@ module HawkularMetrics {
           promises.push(this.HawkularMetric.AvailabilityMetricData(tenantId).query({
             availabilityId: res.id, distinct: true,
             start: 1, end: moment().valueOf()}, (resource) => {
-            res['isUp'] = (resource[0] && resource[0].value === 'up');
+              resource.reverse(); // FIXME: HAWKULAR-366
```

Comment:
IMO, is better that it fails when server changes so we remember to remove the unnecessary operation. Plus sorting would be heavier, for something we know that reverse is what we want.